/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
/home/wtx/anaconda3/envs/dgl/lib/python3.10/site-packages/pytorch_memlab/line_profiler/line_records.py:184: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  merged[byte_cols] = merged[byte_cols].applymap(readable_size)
Killed
./train.sh: 109: cannot create logs/TGAT/gdelt/tgat_gdelt_2layer_500.log: Directory nonexistent
./train.sh: 109: py: not found
Traceback (most recent call last):
  File "/home/wtx/workspace/cpp_project/tgl/train_partition.py", line 57, in <module>
    plan = pg.create_train_plan(3,0.8,False)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 209, in create_train_plan
    train_plan = scheduler.generate_plan()
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 239, in generate_plan
    relation_score = self.compute_relation_score(j, plan)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 222, in compute_relation_score
    print(f"partition {pid} with plan {plan} relation score: {scores:.4f}")
TypeError: unsupported format string passed to list.__format__
Traceback (most recent call last):
  File "/home/wtx/workspace/cpp_project/tgl/train_partition.py", line 57, in <module>
    plan = pg.create_train_plan(3,0.8,False)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 209, in create_train_plan
    train_plan = scheduler.generate_plan()
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 239, in generate_plan
    relation_score = self.compute_relation_score(j, plan)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 222, in compute_relation_score
    print(f"partition {pid} with plan {plan} relation score: {scores:.4f}")
TypeError: unsupported format string passed to list.__format__
Traceback (most recent call last):
  File "/home/wtx/workspace/cpp_project/tgl/train_partition.py", line 57, in <module>
    plan = pg.create_train_plan(3,0.8,False)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 209, in create_train_plan
    train_plan = scheduler.generate_plan()
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 239, in generate_plan
    relation_score = self.compute_relation_score(j, plan)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 222, in compute_relation_score
    print(f"partition {pid} with plan {plan} relation score: {scores:.4f}")
TypeError: unsupported format string passed to list.__format__
Traceback (most recent call last):
  File "/home/wtx/workspace/cpp_project/tgl/train_partition.py", line 57, in <module>
    plan = pg.create_train_plan(3,0.8,False)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 209, in create_train_plan
    train_plan = scheduler.generate_plan()
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 239, in generate_plan
    relation_score = self.compute_relation_score(j, plan)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 222, in compute_relation_score
    print(f"partition {pid} with plan {plan} relation score: {scores:.4f}")
TypeError: unsupported format string passed to list.__format__
Traceback (most recent call last):
  File "/home/wtx/workspace/cpp_project/tgl/train_partition.py", line 57, in <module>
    plan = pg.create_train_plan(3,0.8,False)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 209, in create_train_plan
    train_plan = scheduler.generate_plan()
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 239, in generate_plan
    relation_score = self.compute_relation_score(j, plan)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 222, in compute_relation_score
    print(f"partition {pid} with plan {plan} relation score: {scores:.4f}")
TypeError: unsupported format string passed to list.__format__
Traceback (most recent call last):
  File "/home/wtx/workspace/cpp_project/tgl/train_partition.py", line 57, in <module>
    plan = pg.create_train_plan(3,0.8,False)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 209, in create_train_plan
    train_plan = scheduler.generate_plan()
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 239, in generate_plan
    relation_score = self.compute_relation_score(j, plan)
  File "/home/wtx/workspace/cpp_project/tgl/partition.py", line 222, in compute_relation_score
    print(f"partition {pid} with plan {plan} relation score: {scores:.4f}")
TypeError: unsupported format string passed to list.__format__
1000
